labels.go

```diff
@@ -1,23 +1,50 @@
 package servicefabric
 
-import "strings"
+import (
+	"github.com/containous/traefik/provider/label"
+)
 
-func hasServiceLabel(service ServiceItemExtended, key string) bool {
-	_, exists := service.Labels[key]
-	return exists
+// Must be replace by:
+// func hasFunc(labelName string) func(service ServiceItemExtended) bool {
+// 	return func(service ServiceItemExtended) bool {
+// 		return label.Has(service.Labels, labelName)
+// 	}
```

Comment:
I'll move to this model with the work on #3. As discussed there I'm guessing we're ok to keep the generic functions to allow easy customization in the templates, such as adding custom labels or changing behaviors?

Reply:
I agree but we also need to remove some "logical" from template.
Example:

before:
```
weight = {{getLabelValueWithDefault $service "backend.group.weight" "1"}}
```
after:
```
weight = {{ getWeight $service }}
```